.ruby-version

```diff
@@ -1 +1 @@
-ruby-2.2
\ No newline at end of file
+ruby-2.2
```

Comment:
Support for Ruby 2.2 has ended, maybe set the minimum to 2.3?

https://www.ruby-lang.org/en/news/2018/06/20/support-of-ruby-2-2-has-ended/